### ISSUE TITLE
Localised decimal separator

### ISSUE THREAD

**Issue by rcasula** [RIP-Comm/sossoldi#112]
Use the system default decimal separator instead of the hardcoded `.`.

**Comment by GBergatto** [comment]
Currently, we are using commas and periods inconsistently across the app. The best way to solve this is, as suggested by @rcasula, to read the local of the user. However, in this early stage we could start with the English locale as default, using dot as decimal separator and comma as thousands separator.

**Comment by theperu** [comment]
I agree with @GBergatto since the app is in English I think we should stick with `.` for the decimals and `,` for the thousands

**Comment by Votyer** [comment]
i'll take care of it

**Comment by Votyer** [comment]
Hello everyone, short update from my task.
I changed the method "numToCurrency" in functions class so now it accept the locale as parameter. 
Addictionally I added the flutter lib for string internationalizing(link below)  to support different languages.

I hope to close this task before the end next week

Internationalizing Flutter apps
https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization

**Comment by theperu** [comment]
Great! We'll wait for the PR then!

**Comment by Votyer** [comment]
PR Opened sorry for delay!
